basalt/index.md

```diff
@@ -0,0 +1,205 @@
+---
+layout: default
+title: BASALT Competition 2021
+---
+
+## [Sign-up to participate on <u>[AIcrowd]</u>!](https://www.aicrowd.com/challenges/neurips-2020-minerl-competition)
```

Comment:
This points to the 2020-minerl-competition?

Reply:
Omg good catch, fixed